<ui version="4.0" >
 <class>PropertiesDialog</class>
 <widget class="QDialog" name="PropertiesDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="cursor" >
   <cursorShape>SizeAllCursor</cursorShape>
  </property>
  <property name="windowTitle" >
   <string>Properties</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="fourVertices" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Four Vertices</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="tristate" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="powerups" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="powerupsLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>PowerupSP powerups</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="texture" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mapnameLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map name</string>
   </property>
  </widget>
  <widget class="QLabel" name="maptitleLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map title</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapdescriptionLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map description</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapauthorsLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map authors</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapversionsLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map version_s</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapname" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="maptitle" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapdescription" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapauthors" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapversions" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="configLine" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>280</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="autoselect" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>300</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Auto Select</string>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="tristate" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="nomodify" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>300</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Read Only</string>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="tristate" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="autonotexture" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>320</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="text" >
    <string>Auto No Texture</string>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="tristate" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textureLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>texture</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="level" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="levelLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>level</string>
   </property>
  </widget>
  <widget class="QLabel" name="mapversionLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>240</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>map version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mapversion" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>name</string>
   </property>
  </widget>
  <widget class="QLabel" name="targetNameLabel" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor" >
    <cursorShape>SizeAllCursor</cursorShape>
   </property>
   <property name="text" >
    <string>targetName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="targetName" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="configLine_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>140</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
